### PR TITLE
founders programme slides

### PR DESCRIPTION
**Pull request by sofer**
@charlielafosse slides for this afternoon.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -9,3 +9,4 @@ Slides
 - [Getting Started on Codewars](/slides/codewars-intro.html)
 - [Tech for Better Discovery Workshop](/slides/tfb-discovery-workshop.html)
 - [Slides for non-FAC people to showcase FAC (help me with name)](slides/arsenal-slides.html)
+- [Founders programme intro for students](/slides/founders-programme)
```
Comment (charlielafosse) [inline index.md:12]:
```suggestion
- [Founders programme intro for students](/slides/founders-programme.html)
```
missed out the extension. I think you can hit commit suggestion and it'll go straight in

Comment (charlielafosse) [reply index.md:12]: @sofer

**Review by charlielafosse — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (sofer) [reply index.md:12]: @charlielafosse It shouldn't need it. See e.g. http://facresources.com/slides/tfb

Comment (charlielafosse) [reply index.md:12]: Oh yeah, my bad

**Pull request merged**